### PR TITLE
update script for datahub add header

### PR DESCRIPTION
**Pull request by jeanmi151**
wrong header for the two script of datahub to add header

### CODE REVIEW

File: datahub/scripts/add-header-webcomponent.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/ash
```
Comment (jahow) [inline datahub/scripts/add-header-webcomponent.sh:1]: I would use `sh` here, I feel it would be safer

**Review by jahow — APPROVED** [review]
Thanks! I'm approving but I would have personally used `sh` instead of `ash`, I feel like `sh` is really the common denominator for shell runtimes but maybe I'm wrong...?

**Comment by edevosc2c** [comment]
@jeanmi151 did you check if it works with sh though? sh doesn't provide the same functionalities as ash or bash

**Comment by jeanmi151** [comment]
always testing before merge ...

**Pull request merged**